extensions/wikia/Recirculation/templates/client/rail.mustache

```diff
@@ -1,13 +1,14 @@
-<section class="recirculation-rail module">
-	<h2>{{ title }}</h2>
+<section class="recirculation-rail module group-{{ group }}">
+	<h2>{{{ title }}}</h2>
```

Comment:
It's just for the `<br>`. Another option would be to modify the title post render, which I'm not really a fan of either. Suggestions are appreciated :)